Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py

```diff
@@ -58,7 +58,7 @@
     if platform.system() == 'Windows':
         AutoInstall.register(Package('win_inet_pton', Version(1, 1, 0), pypi_name='win-inet-pton'))
 
-if sys.version_info >= (3, 12):
+if sys.version_info >= (3, 13):
```

Comment:
I am not sure about this fix by myself. setuptools-68.1.2 might be a better choice for python 3.11 or higher since `git-webkit pr` is also broken in python 3.12 with setuptools-56.0.0.

@mcatanzaro, does the beautifulsoup4 still need the old setuptools? Can we bump up setuptools to 68.1.2?

Reply:
Why are you checking for Python 3.13? The latest release currently is 3.12.

Reply:
This change will reintroduce https://bugs.webkit.org/show_bug.cgi?id=260726 so it's not a good solution.

> @mcatanzaro, does the beautifulsoup4 still need the old setuptools?

I don't know about beautifulsoup4.

> Can we bump up setuptools to 68.1.2?

Not for python 3.11, because that would reintroduce https://bugs.webkit.org/show_bug.cgi?id=260889

Reply:
> > @mcatanzaro, does the beautifulsoup4 still need the old setuptools?
> 
> I don't know about beautifulsoup4.

You mentioned beautifulsoup4 at https://github.com/WebKit/WebKit/pull/17214, so I thought it was a blocker of bumping up Python.